Misc/NEWS.d/next/Library/2022-05-08-18-51-14.gh-issue-89336.TL6ip7.rst

```diff
@@ -0,0 +1,3 @@
+The ``configparser`` module APIs deprecated to be removed in 3.12 have been
+removed: The ``SafeConfigParser`` class alias, the ``ParsingError.filename``
+property, and the ``ConfigParser.readfp`` method have all been removed.
```

Comment:
Perhaps?

```suggestion
Removed ``configparser`` module APIs: 
the ``SafeConfigParser`` class alias, the ``ParsingError.filename``
property, and the ``ConfigParser.readfp`` method, deprecated since Python 3.2.
```